distributed/worker.py

```diff
@@ -2082,10 +2082,10 @@ def transition_waiting_ready(
     def transition_cancelled_error(
         self,
         ts: TaskState,
-        exception,
-        traceback,
-        exception_text,
-        traceback_text,
+        exception: to_serialize,
```

Comment:
nit: I think I would prefer using the `Serialize` itself for type annotations and not the alias just for consistency purposes

Reply:
I'll change it to Serialize everywhere. I used to_serialize as I found it like that elsewhere already; I don't understand its purpose to begin with?